lib/salestation/app/input_verification.rb

```diff
@@ -6,12 +6,27 @@ module InputVerification
       def verify_input(schema)
         -> (request) do
           result = schema.call(request.input)
-          if result.success?
-            request.replace_input(result.output)
+
+          dry_validation_version = Gem.loaded_specs['dry-validation'].version
+          if dry_validation_version < Gem::Version.new('1.0')
```

Comment:
shouldn't this be `< Gem::Version.new('1.7')` as previously we expected dry validation to be `1.6`?

Reply:
The breaking change is across v1.0, not v1.6. idk why it wasn't changed before on the other upgrades, maybe the other services didn't use salestation like how engagement-api does. I only bumped dry-validation to fix a warning in the project.